aws-logs-loggroup/aws-logs-loggroup.json

```diff
@@ -3,8 +3,14 @@
     "description": "Resource schema for AWS::Logs::LogGroup",
     "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs.git",
     "properties": {
+        "KmsKeyId": {
+            "description": "The Amazon Resource Name (ARN) of the CMK to use when encrypting log data.",
+            "type": "string",
+            "maxLength": 256,
+            "pattern": "^arn:[a-z0-9-]+:kms:[a-z0-9-]+:\\d{12}:(key|alias)/.+\\Z"
```

Comment:
Will want to come back to this so we can $ref an `AWS::KMS::Key` resource's `Arn` property when that's available.